src/licensedcode/data/rules/proprietary-license_519.RULE

```diff
@@ -0,0 +1,19 @@
+[Triangle] may be freely redistributed under the
```

Comment:
We should remove the leading and trailing unique names as thy are making the index heavier but do not help with detection.

```suggestion
may be freely redistributed under the
```